contracts/chain-adapters/Linea_Adapter.sol

```diff
@@ -0,0 +1,354 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
I would suggest adding a few simple tests, like we have for the other adapters.

Reply:
done here https://github.com/across-protocol/contracts-v2/pull/382/commits/32d72c03d07f6ffaba0101fd3e5d2890931f5cc1